backend/tests/api/test_openapi_responses.py

```
@@ -1423,6 +1908,39 @@ def test_get_response_success(
+        response = test_client.get(
+            f"/api/v1/responses/resp_{test_task.id}",
+            headers={"X-API-Key": test_api_key[0]},
+        )
+
+        assert response.status_code == 200
+        data = response.json()
+        assert data["id"] == f"resp_{test_task.id}"
+        assert data["status"] == "failed"
+        assert data["error"] == {
+            "code": "PROMPT_PROTECTION_BLOCKED",
+            "message": "该请求无法处理，请调整问题后再试。",
```

Ruff (0.16.3)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)